data/strings/types_strings.txt

```diff
@@ -2517,6 +2517,68 @@
     sk = Zberné suroviny
     fa = مرکز بازیافت
 
+  [type.amenity.recycling_container]
+    en = Recycling container
+    es = Contenedor de reciclaje
+    de = Recyclingbehälter
+    ru = Контейнер для вторсырья
```

Comment:
copy the language order from an older translation, please

Reply:
done

Reply:
ping on this comment

Reply:
sorry, somehow I missed this. fixed now